### ISSUE TITLE
Error when trying to import xml on bpmnjs

### ISSUE THREAD

**Issue by juanferrari** [bpmn-io/diagram-js#586]
**Describe the Bug**

I am using bpmn-js (8.8.2) and I noticed that after the last upgrade of diagram-js: 7.6.0 -> 7.6.1, diagrams didn't render anymore when trying to import a xml on the Modeler.
Downgrading to 7.6.0 fixes the issue.

I am getting the following error on my console:

```

Uncaught (in promise) Error: layer <base> already created at index <-10>
    at Canvas.getLayer (Canvas.js?c639:294)
    at Canvas.createPlane (Canvas.js?c639:370)
    at Canvas.setRootElement (Canvas.js?c639:608)
    at BpmnImporter.add (BpmnImporter.js?6491:112)
    at Object.root (Importer.js?e00c:53)
    at visitRoot (BpmnTreeWalker.js?896b:83)
    at Object.handleDefinitions (BpmnTreeWalker.js?896b:215)
    at render (Importer.js?e00c:69)
    at eval (Importer.js?e00c:80)
    at new Promise (<anonymous>)
```

**Steps to Reproduce**

Import XML using BPMN Modeler.

 - Browser: [Chrome 95]
 - OS: [ Windows 10]
 - Library version: [7.6.1]

**Comment by nikku** [comment]
Thanks for reporting. We can [reproduce it](https://github.com/bpmn-io/bpmn-js/pull/1526) and will look into a fix as soon as possible.

**Comment by nikku** [comment]
To be tackled next week. CC @marstamm

**Comment by nikku** [comment]
Fixed with `v7.6.2` (reverted the broken feature).
